homeassistant/components/airzone/select.py

```diff
@@ -41,14 +41,14 @@ class AirzoneSelectDescription(SelectEntityDescription, AirzoneSelectDescription
 
 
 GRILLE_ANGLE_DICT: Final[dict[str, int]] = {
-    "90º": GrilleAngle.DEG_90,
-    "50º": GrilleAngle.DEG_50,
-    "45º": GrilleAngle.DEG_45,
-    "40º": GrilleAngle.DEG_40,
+    "90deg": GrilleAngle.DEG_90,
```

Comment:
Can make this a list instead with the `GrillAngle` values

Reply:
Should be fixed with e503a1 93f8bd